scripts/cargo_sort/cargo_sort.py

```diff
@@ -347,7 +347,8 @@ def run_dprint_fmt(directory):
     internal_crates_dict = get_package_names_from_cargo_tomls(args.target)
     
     # add special cases
-    internal_crates_dict["iota-rust-sdk"] = None
+    internal_crates_dict["iota-sdk-types"] = None
```

Comment:
may want to also handle iota-sdk since we'll probably end up using that instead

Reply:
I told Max to only add the 2 crates we are presently using in the monorepo and that we would add more ourselves when/if needed. I don't mind them being preventively added if you prefer that.